src/Modules/CalcDefence.lua

```diff
@@ -1707,6 +1728,16 @@ function calcs.defence(env, actor)
 						frostShield = frostShield - tempDamage
 						Damage[damageType] = Damage[damageType] - tempDamage
 					end
+					-- soul link is not implemented for now
+					if soulLink > 0 then
+						local tempDamage = m_min(Damage[damageType] * output["SoulLinkMitigation"] / 100, soulLink)
+						soulLink = soulLink - tempDamage
+						Damage[damageType] = Damage[damageType] - tempDamage
+					end
+					-- frost shield and soul link does not count as you taking damage
+					if DamageIn["TrackLifeLoss"] then
+						output[damageType.."LifeLossLost"] = output[damageType.."LifeLossLost"] + Damage[damageType]
```

Comment:
```suggestion
					if DamageIn["TrackDamageTaken"] then
						output[damageType.."DamageTaken"] = output[damageType.."DamageTaken"] + Damage[damageType]
```
This has nothing to do with life using this naming is confusing.